networkx/algorithms/bipartite/covering.py

```diff
@@ -0,0 +1,52 @@
+#    Copyright 2016 NetworkX developers.
+#    Copyright (C) 2004-2016 by
+#    Nishant Nikhil <nishantiam@gmail.com>
+#    All rights reserved.
+#    BSD license.
+
+""" Functions related to graph covers."""
+
+import networkx as nx
+from networkx.utils import not_implemented_for, arbitrary_element
+from networkx.algorithms.bipartite.matching import hopcroft_karp_matching
+from networkx.algorithms.covering import min_edge_cover as _min_edge_cover
+
+
+@not_implemented_for('directed')
+@not_implemented_for('multigraph')
+def min_edge_cover(G, matching_algorithm=None):
+    r"""Returns a set of edges which constitutes
+    the minimum edge cover of the graph.
+
+    A smallest edge cover can be found in polynomial time by finding
+    a maximum matching and extending it greedily so that all vertices
+    are covered.
+
+    Parameters
+    ----------
+    G : An undirected, unweighted bipartite graph.
```

Comment:
This should probably be something like

    G : NetworkX graph
        An undirected bipartite graph.

The part after the colon is the "type" of the argument. The "unweighted" is irrelevant.